straightline.py

```diff
@@ -174,14 +244,12 @@ def run(self):
             drive=self.drive,
             dict=self.dict
         )
-        logger.info('Wait ...')
-        time.sleep(2)
         logger.info('Setting up image capture thread')
         self.image_capture_thread = ImageCapture(
             camera=self.camera,
             processor=self.processor
         )
-        pygame.mouse.set_visible(False)
+        pygame.mouse.set_visible(True)
```

Comment:
is this still true? do you still need the cursor visible?

Reply:
cursor is now used to indicate perceived line or marker position